api.swagger.yaml

```diff
@@ -2370,6 +2370,517 @@ paths:
+                      - $ref: '#/components/schemas/grade-schema'
+      description: 'Search for grades. An empty array is returned if no grades are found.'
+  /grades/{grade_id}:
+    get:
+      tags:
+        - grades
+      parameters:
+        - description: The Invotra UUID of the grade.
+          in: path
+          name: grade_id
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: Returns a single complete grade object.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/grade-schema'
+        '400':
+          description: The supplied UUID was malformed.
+        '404':
+          description: The UUID does not correspond to an existing grade.
+      description: Get information about the grade.
+    put:
+      tags:
+        - grades
+      parameters:
+        - description: The Invotra UUID of the grade.
+          in: path
+          name: grade_id
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: The grade information was updated successfully.
+        '400':
+          description: The supplied UUID or JSON was malformed.
+        '404':
+          description: The UUID does not correspond to an existing grade.
+        '422':
+          description: |
+            The grade was not updated due to a Drupal validation failure or a
+            JSON schema validation failure.
+      description: Update grade information.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/grade-schema'
+        required: true
+    delete:
+      tags:
+        - grades
+      description: Delete a grade
+      parameters:
+        - description: The Invotra UUID of the grade.
+          in: path
+          name: grade_id
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: The grade was successfully deleted.
+        '400':
+          description: The supplied UUID was malformed.
+        '404':
+          description: The UUID does not correspond to an existing grade.
+  /organisational_skills:
+    post:
+      tags:
+        - organisational skills
+      responses:
+        '201':
+          description: The organisational skill was created successfully.
+          headers:
+            Location:
+              description: |
+                The URL at which the newly provisioned organisational skill may be retrieved.
+              schema:
+                type: string
+        '400':
+          description: 'The organisational skill was not saved due to malformed JSON.'
+        '403':
+          description: 'User is not authorized to create an organisational skill.'
+        '409':
```

Comment:
Just talked this through with product. Let's not add the 409 error/validation yet, sorry about that.

Reply:
took the 409 out of /organisational_skills